src/libraries/System.Drawing.Common/src/System/Drawing/macFunctions.cs

```diff
@@ -211,14 +211,14 @@ internal static void ReleaseContext(IntPtr port, IntPtr context)
         #region Cocoa Methods
         [DllImport("libobjc.dylib")]
         public static extern IntPtr objc_getClass(string className);
-        [DllImport("libobjc.dylib")]
-        public static extern IntPtr objc_msgSend(IntPtr basePtr, IntPtr selector, string argument);
-        [DllImport("libobjc.dylib")]
-        public static extern IntPtr objc_msgSend(IntPtr basePtr, IntPtr selector);
-        [DllImport("libobjc.dylib")]
-        public static extern void objc_msgSend_stret(ref Rect arect, IntPtr basePtr, IntPtr selector);
         [DllImport("libobjc.dylib", EntryPoint = "objc_msgSend")]
-        public static extern bool bool_objc_msgSend(IntPtr handle, IntPtr selector);
+        public static extern IntPtr get_currentContext(IntPtr basePtr, IntPtr selector);
+        [DllImport("libobjc.dylib", EntryPoint = "objc_msgSend")]
+        public static extern IntPtr get_graphicsPort(IntPtr basePtr, IntPtr selector);
+        [DllImport("libobjc.dylib", EntryPoint = "objc_msgSend_stret")]
+        public static extern void get_bounds(ref Rect arect, IntPtr basePtr, IntPtr selector);
+        [DllImport("libobjc.dylib", EntryPoint = "objc_msgSend")]
+        public static extern bool get_isFlipped(IntPtr handle, IntPtr selector);
```

Comment:
What is size of `bool` in Objective-C - does it match what the marshaler is going to use here?

Reply:
It is not. This is a bug. I'll fix it tomorrow.

Reply:
Is the "gist" linked from https://docs.microsoft.com/en-us/xamarin/ios/internals/objective-c-selectors buggy too then?

Reply:
Yes, and it will cause errors on MacOS arm64.